builtin/mainmenu/pkgmgr.lua

```diff
@@ -639,6 +640,7 @@ function pkgmgr.install_dir(type, path, basename, targetpath)
 	end
 
 	-- Copy it
+	core.delete_dir(targetpath)
```

Comment:
In some cases you're now deleting the directory twice.

```Lua
			core.delete_dir(targetpath)
			core.create_dir(targetpath)
```
This repeated code above could be moved here.